src/pages/cli/graphql/custom-business-logic.mdx

```diff
@@ -58,16 +58,72 @@ The Amplify CLI provides support for maintaining multiple environments. When you
 
 </Block>
 
-<Block name="Function deployed without Amplify CLI">
+<Block name="AWS CDK">
 
-If you deployed your Lambda function without Amplify CLI then you must provide the full Lambda function name in the `name` parameter. If you deployed the same function with the name echofunction then you would have:
+First, create your Lambda function in CDK with your logic and set the `functionName` parameter.
+
+```ts
+const echoLambda = new lambda.Function(this, 'EchoLambda', {
+  functionName: 'EchoLambda', // MAKE SURE THIS MATCHES THE @function's "name" PARAMETER
+  code: lambda.Code.fromAsset(path.join(__dirname, 'handlers/echo')),
+  handler: 'index.handler',
+  runtime: lambda.Runtime.NODEJS_18_X
+})
+
+const amplifyApi = new AmplifyGraphqlApi(this, 'AmplifyCdkGraphQlApi', {
+  definition: AmplifyGraphqlDefinition.fromFiles(path.join(__dirname, 'schema.graphql')),
+  authorizationModes: {
+    defaultAuthorizationMode: 'API_KEY',
+    apiKeyConfig: {
+      expires: cdk.Duration.days(30) 
+    }
+  },
+})
+```
+
+To connect an AWS Lambda resolver to the GraphQL API, add the `@function` directive to a field in your GraphQL schema.
 
 ```graphql
 type Query {
   echo(msg: String): String @function(name: "echofunction")
 }
 ```
 
+Optionally, if you don't want to hard-code the function name into the GraphQL schema, you can also set an arbitrary name in the GraphQL schema and then map a function within CDK to the function name.
+
+```ts
+const coolLambdaFunction = new lambda.Function(this, 'MyCoolLambdaFunction', {
+  functionName: 'MyCoolLambdaFunction', // MAKE SURE THIS MATCHES THE @function's "name" PARAMETER
```

Comment:
Can remove the 'MAKE SURE' bit from this example (or even let it just be auto-generated.